ably/realtime_conn.go

```diff
@@ -470,9 +470,9 @@ func (c *Connection) Key() string {
 //
 // Ping returns non-nil error without any attempt of communication with Ably
 // if the connection state is ConnectionStateClosed or ConnectionStateFailed.
-func (c *Connection) Ping() (ping, pong time.Duration, err error) {
-	return 0, 0, errors.New("TODO")
-}
+//func (c *Connection) Ping() (ping, pong time.Duration, err error) {
+//	return 0, 0, errors.New("TODO")
+//}
```

Comment:
Is there some difficulty in actually implementing this? I think we should either implement it or remove the code rather than just commenting it out.

Reply:
Actually, we need to implement it, but it's not prioritized for this release. #263

Reply:
Yes we are supposed to be implement it #263 , but it's of less priority as of now

Reply:
Ok, so I don't see the benefit of commenting out this code rather than just leaving it as a TODO until it gets implemented?

Reply:
Current TODO just returns error, we don't want it to be a part of public interface, at the same time, we don't want to 
lose current partial implementation/ doc-comments for the method. This is going to be implemented soon after beta release.